third_party/blink/renderer/core/brave_session_cache.cc

```diff
@@ -45,7 +46,7 @@ float ConstantMultiplier(double fudge_factor, float value, size_t index) {
 
 float PseudoRandomSequence(uint64_t seed, float value, size_t index) {
   static uint64_t v;
-  const double maxUInt64AsDouble = UINT64_MAX;
+  const double maxUInt64AsDouble = static_cast<double>(UINT64_MAX);
```

Comment:
not sure if it matters, but this is not necessarily correct. Not all integers above 2^53-1 can be represented accurately in a double

Reply:
@pilgrim-brave Do you know what we should do with this line? I think the linter suggested I add the static_cast<double>, but I'm not sure how the math is supposed to work here.

Reply:
On @pilgrim-brave 's suggestion, I am now using
```
static const double maxUInt64AsDouble = 18446744073709551615.0;
```
As seen here: https://source.chromium.org/search?q=maxUInt64AsDouble&sq=&ss=chromium%2Fchromium%2Fsrc